### PR TITLE
SW-2518 Apply correct border color to fieldset

### PR DESCRIPTION
**Pull request by alexTerraformation**
This was the simplest solution to fix the issue described in SW-2518.  I tried to clean up some of the styles for these components and found quite a tangle that wasn't worth it to work through right now.

### CODE REVIEW

File: src/components/Autocomplete/styles.scss [hunk 1]
```diff
@@ -49,6 +49,9 @@
     height: '21px';
   };
   & .MuiOutlinedInput-root {
+    fieldset {
```
Comment (karthikbtf) [inline src/components/Autocomplete/styles.scss:52]: definitely simpler 🎉 thanks!!
i think we may need a similar change for the error condition.
if it helps, you can play around in storybook (the error text is settable in the ui)

**Review by charlie-tf — APPROVED** [review]
👍

**Review by constanzauanini — APPROVED** [review]

File: src/components/Autocomplete/styles.scss (hunk 1 above)
Comment (alexTerraformation) [reply src/components/Autocomplete/styles.scss:52]: Yes I missed the error border on this one, thanks!

Comment (karthikbtf) [reply src/components/Autocomplete/styles.scss:52]: Looks good 👍 thanks!

**Review by karthikbtf — APPROVED** [review]
Lgtm!

**Pull request merged**